From f5651a4f67db8f9bd0a256fd36b26466085fee7d Mon Sep 17 00:00:00 2001
From: JP Abgrall <jpa@google.com>
Date: Fri, 7 Jun 2013 15:23:31 -0700
Subject: [PATCH] android: configs: no MODULES for base, no SIP for
 recommended

We don't like CONFIG_MODULES anymore.
Connection tracker handling of large SIP fails.

Change-Id: Ie3c65aefcc6181752d6656c97e63035e5b5653ff
Signed-off-by: JP Abgrall <jpa@google.com>
---
 android/configs/android-base.cfg        | 1 +
 android/configs/android-recommended.cfg | 1 +
 2 files changed, 2 insertions(+)

diff --git a/android/configs/android-base.cfg b/android/configs/android-base.cfg
index 1969f30..2b98436 100644
--- a/android/configs/android-base.cfg
+++ b/android/configs/android-base.cfg
@@ -1,4 +1,5 @@
 # CONFIG_INET_LRO is not set
+# CONFIG_MODULES is not set
 # CONFIG_OABI_COMPAT is not set
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
diff --git a/android/configs/android-recommended.cfg b/android/configs/android-recommended.cfg
index 175f7c2..ca3cc86 100644
--- a/android/configs/android-recommended.cfg
+++ b/android/configs/android-recommended.cfg
@@ -5,6 +5,7 @@ CONFIG_COMPACTION=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 # CONFIG_INPUT_MOUSE is not set
 # CONFIG_LEGACY_PTYS is not set
+# CONFIG_NF_CONNTRACK_SIP is not set
 # CONFIG_PM_WAKELOCKS_GC is not set
 # CONFIG_VT is not set
 CONFIG_ANDROID_RAM_CONSOLE=y
-- 
1.8.0

